### ISSUE TITLE
feat: ✨ add defined aspect ratios to Tailwind config

### ISSUE THREAD

**Issue by karlbaumhauer** [solid-design-system/solid#728]
## User Story
As a user of the Solid Design System, I would like to use pre-set aspect ratios for any kind of media (e. g. `<video>` or `<img>`). These should be available when using TailwindCSS (e. g. `aspect-square`, `aspect-golden`, `aspect-3/2`) or SCSS (e. g. `$aspectRatioSquare` etc.).

The created tokens should then be added to the tokens page from #355.

## Documentation

The following list all needed ratios: 
[Figma Component](https://www.figma.com/file/VTztxQ5pWG7ARg8hCX6PfR/Solid-DS-%E2%80%93-Component-Library?type=design&node-id=1171-22142&mode=design&t=VxS2a2hyTlbIlNBO-11)
[Figma Dokumentation](https://www.figma.com/file/8u2hIRmhVX3mjW8462Au5I/Image?type=design&node-id=0-1&mode=design)
[Aspect Ratio CSS Property – Figma Helper Component
](https://www.figma.com/file/g67ZBA2gth8so2izvNOD7m/sd-aspect-ratio?type=design&node-id=19229-5703&mode=design)

Here is how you can define own aspect-ratios:

https://tailwindcss.com/docs/aspect-ratio#customizing-your-theme

### Needed Ratios => pls double check with the Figma links above

- square
- 6/5
- 5/4
- 4/3
- 3/2
- 16/10
- golden-ratio (try to use calc to get the right ratio)
- 2/1
- 21/9
- 3/4
- 4/5

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Make sure that SCSS export works correctly
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by karlbaumhauer** [comment]
@coraliefeil I was not able to find a hand-off in figma. Is it still missing? If yes, do we need a design ticket for this?

**Comment by coraliefeil** [comment]
@karlbaumhauer I’ve updated the links

**Comment by karlbaumhauer** [comment]
> @karlbaumhauer I’ve updated the links

thx.

2 questions:
- what exactly is the "Mastercomponent" (as we never had this before)?
- what about the brandshape image? can we exclude it or is it even part of the image component? if not part and needs to stay in there, I would explicitly mention to ignore it in the ticket as it might confuses while working on this ticket.

**Comment by karlbaumhauer** [comment]
> * what exactly is the "Mastercomponent" (as we never had this before)?

=> just a naming for Figma => will be added to contribution guide.

> * what about the brandshape image? can we exclude it or is it even part of the image component? if not part and needs to stay in there, I would explicitly mention to ignore it in the ticket as it might confuses while working on this ticket.

=> excluded from this ticket => new ticket will be created to check how we can solve this technically in development (current solution is actually not the 100% correct brandshape).

### Edit from pre-refinement:
- No CSS style but added to our tailwind config => @mariohamann pls reformulate the description
- brandshape excluded for now => follow up ticket will be created.

**Comment by coraliefeil** [comment]
I moved former sd-image-placeholder to CSS section and renamed it to sd-aspect-ratio
[Aspect Ratio CSS Property – Figma Helper Component](https://www.figma.com/file/g67ZBA2gth8so2izvNOD7m/sd-aspect-ratio?type=design&node-id=19229-5703&mode=design)

Now we just need to sort out sd-image/sd-media/sd-video ... which is for what and do we need to deprecate sth.

**Comment by karlbaumhauer** [comment]
blocked by #355
